extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -32,26 +32,24 @@ export const DrawerState = {
 
 /**
  * Drawer's template.
- * @param {!Element} element
  * @return {!Element}
  */
-const getTemplateEl = (element) => {
-  return htmlFor(element)`
+const renderDrawerElement = () => {
+  return (
```

Comment:
Could we use implicit returns in places like this?

Reply:
For sure, but I meant to leave outer structure intact. I'll leave this style choice for the Stories team to handle later.